rootless.md

```diff
@@ -0,0 +1,31 @@
+# Shortcomings of Rootless Podman
+
+* Lacking “How To” documentation or documentation in general
+* If /etc/subuid and /etc/subgid not setup for a user, then podman commands
+can easily fail
+  * This can be a big issue on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
+  * We are working to get support for NSSWITCH on the /etc/subuid and /etc/subgid files.
+* Podman can not create containers that bind to ports < 1024.
+  * The kernel does not allow processes without CAP_NET_BIND_SERVICE to bind to low ports.
+* No cgroup Support (hopefully fixed when cgroups V2 happens).
+  * Cgroups V1 does ont support cgroup delegation.
+* Can not share container images with CRI-O or other users
```

Comment:
this would be very nice to have: if one could easily push an image from rootless store to the root store

Reply:
Does it still share the images with rootless CRI-O?

Reply:
Container/storage for a rootless store would be shared between all users of it,  Podman, CRI-O, Buildah and soon Skopeo.

Reply:
@TomasTomecek We would need to have a way to increase privileges.  Perhaps once Skopeo gets support for rootless containers would could have a way to run a sudo command on skopeo
Something like
```
sudo skopeo copy containers-storage:~dwalsh/Image containers-storage
```

Reply:
Dan, that's exactly what I have in mind. The way I personally bypass this limitation now is that I push the image to dockerd. I think pushing to a file could also be a good middleman.